src/gui_common/art_gallery/Slidescreen.cs

```diff
@@ -0,0 +1,447 @@
+        var instanced = scene.Instance();
+
+        modelHolder.AddChild(instanced);
+
+        var mesh = instanced.GetNode<MeshInstance>(item.Asset.MeshNodePath!);
+        var minDistance = mesh.GetTransformedAabb().Size.Length();
+        var maxDistance = PhotoStudio.CameraDistanceFromRadiusOfObject(minDistance);
+
+        modelViewerCamera.MinCameraDistance = minDistance;
+        modelViewerCamera.MaxCameraDistance = maxDistance;
+        modelViewerCamera.Distance = maxDistance;
+    }
+
+    private void UpdatePlayback()
+    {
+        var item = Items?[currentSlideIndex] as GalleryCardAudio;
+
+        if (playbackBar == null || fullscreenTextureRect == null)
+            return;
+
+        if (item?.Asset.Type != AssetType.AudioPlayback)
+        {
+            playbackBar.Hide();
+            playbackBar.AudioPlayer = null;
+            return;
+        }
+
+        playbackBar.DisconnectSignals(nameof(PlaybackBar.Started));
+        playbackBar.DisconnectSignals(nameof(PlaybackBar.Stopped));
+        playbackBar.Connect(nameof(PlaybackBar.Started), item, "OnStarted");
+        playbackBar.Connect(nameof(PlaybackBar.Stopped), item, "OnStopped");
+
+        playbackBar.AudioPlayer = item.Player;
+        playbackBar?.Show();
+
+        // TODO: Temporary until there's a proper "album" art for audios
+        fullscreenTextureRect.Texture = item.MissingTexture;
+    }
+
+    private void UpdateHandles()
+    {
+        if (toolbar == null || closeButton == null)
+            return;
+
+        toolbar.Visible = handlesVisible;
+        closeButton.Visible = handlesVisible;
+    }
+
+    private void OnSlideFaded(Object @object, NodePath key)
+    {
+        _ = @object;
+        _ = key;
+
+        UpdateScreen();
+        slideshowTween.InterpolateProperty(fullscreenTextureRect, "modulate", null, Colors.White, 0.5f);
+        slideshowTween.Start();
+    }
+
+    private void OnScaledUp(Object @object, NodePath key)
+    {
+        _ = @object;
+        _ = key;
+
+        HandlesVisible = true;
+        FullRect = true;
+        BoundToScreenArea = true;
+    }
+
+    private void OnScaledDown(Object @object, NodePath key)
+    {
+        _ = @object;
+        _ = key;
+
+        HandlesVisible = true;
+        Hide();
+    }
+
+    private void OnNextSlideButtonPressed()
+    {
+        GUICommon.Instance.PlayButtonPressSound();
+        AdvanceSlide();
+    }
+
+    private void OnPreviousSlideButtonPressed()
+    {
+        GUICommon.Instance.PlayButtonPressSound();
+        RetreatSlide();
+    }
+
+    private void OnSlideshowModeButtonToggled(bool pressed)
+    {
+        GUICommon.Instance.PlayButtonPressSound();
+        SlideshowMode = pressed;
+    }
+
+    private void OnCloseButtonPressed()
+    {
+        CustomHide();
+    }
+
+    private void OnCloseButtonUpdate()
+    {
+        var icon = closeButton!.GetChild<TextureRect>(0);
+
+        if (closeButton?.GetDrawMode() == BaseButton.DrawMode.Pressed)
```

Comment:
You already null overrode on the above line so I think the suppression from here is pretty much useless and shouldn't be here.